types/instance.d.ts

```diff
@@ -30,6 +30,8 @@ export interface PrintRoutesOptions {
   includeHooks?: boolean
 }
 
+type AsyncFunction = (...args: any) => Promise<any>;
```

Comment:
Should we also support `PromiseLike<any>`?

Reply:
Well, we define the functions as async. So they should be Promise and not promiselike. But i am open for it. Maybe fix this first than we can think about optimizing it.